userguide/snippets/alertevents.rst

```diff
@@ -20,57 +20,56 @@ the system is warning that the S.M.A.R.T. service is not running.
    Example Alert Message
 
 
-Informational messages have a green :guilabel:`OK`, warning messages
-flash yellow, and messages requiring attention are listed as a red
-:guilabel:`CRITICAL`. CRITICAL messages are also emailed to the root
+Alert messages have a blue and white :literal:`i` icon. Messages
+identified as CRITICAL in the middleware are also emailed to the root
```

Comment:
Do we describe what the middleware is anywhere?  We can simplify this:

"Critical messages are emailed to the root"